### ISSUE TITLE
Date missing from export

### ISSUE THREAD

**Issue by asbjornu** [labocho/movable_type_format#2]
Thanks again for this wonderful tool! After close inspection it seems like the `DATE` is missing from the main blog entries. Comments have `DATE`, though. Here's an example of a blog entry in a Movable Type export file which `DATE` field is not converted to JSON:

```
AUTHOR: asbjorn
TITLE: Title
STATUS: Publish
ALLOW COMMENTS: 0
CONVERT BREAKS: textile_1
ALLOW PINGS: 0

DATE: 06/23/2003 11:34:22 PM
-----
BODY:
...
```

**Comment by asbjornu** [comment]
I've tried to add the `DATE` field to the test inside `parser_spec.rb` and then do:

```ruby
one.date.should == Time.parse("2003-06-23 23:34:22 +0200")
```

afterwards, but it seems to go through the parsing just fine, so perhaps it's just a problem with the conversion to JSON? I'm not sure where to look to start fixing this, so a pointer (or a commit that fixes the problem) would be highly appreciated! 😄

**Comment by labocho** [comment]
This problem is probably caused by a blank line before `DATE` field in the example. [The documentation of format](https://movabletype.org/documentation/appendices/import-export-format.html) does not mention about such a blank line in metadata.

But in COMMENT / PING section,

> Any line that does not match one of the above keys starts the body of (the comment|the ping excerpt)

A blank line in COMMENT / PING metadata, probably indicate beginning of body.

I will inspect MovableType import/export behavior. But it takes time. Please remove blank lines in metadata until fix it.

By the way, is your file generated by MovableType? What version?

**Comment by asbjornu** [comment]
I see. I just found it strange that I weren't able to reproduce the problem in any of the tests. The export is generated by Movable Type, but I can't remember which version. The export is many years old.

**Comment by asbjornu** [comment]
I tried to adjust the blank lines around `DATE`, but just ended up with the following exception:

```ruby
/usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/lib/movable_type_format/field.rb:119:in `initialize': min out of range (ArgumentError)
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/lib/movable_type_format/field.rb:119:in `new'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/lib/movable_type_format/field.rb:119:in `parse_date'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/lib/movable_type_format/field.rb:45:in `value'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/lib/movable_type_format/section/base.rb:30:in `block (2 levels) in define_field_accessors'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/lib/movable_type_format/entry.rb:47:in `block (2 levels) in <class:Entry>'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/lib/movable_type_format/entry.rb:22:in `block in serialize'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/lib/movable_type_format/entry.rb:19:in `each'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/lib/movable_type_format/entry.rb:19:in `serialize'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/exe/mt2json:25:in `block in <top (required)>'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/lib/movable_type_format/parser.rb:25:in `parse'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/exe/mt2json:25:in `each'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/exe/mt2json:25:in `map'
	from /usr/local/lib/ruby/gems/2.4.0/gems/movable_type_format-0.1.2/exe/mt2json:25:in `<top (required)>'
	from /usr/local/bin/mt2json:22:in `load'
	from /usr/local/bin/mt2json:22:in `<main>'
```

**Comment by asbjornu** [comment]
Okay, I figured out the problem with the date parsing and submitted #5. My export now works, but that is after removing the blank lines before `DATE`, which might be good to have fixed even though I won't have use for it. Thanks!

**Comment by labocho** [comment]
I found [the MovableType code](https://github.com/movabletype/movabletype/blob/1585520da1bc4d398d34c734fddaff3277fc2187/lib/MT/ImportExport.pm#L124) that ignores empty line in metadata.  And I fixed my parser at 20ec23a903aa51fcd98d9e7c5fd48aa4ae53a106. Thanks!

**Comment by asbjornu** [comment]
Excellent, thanks!
